### DISCUSSION TITLE
Upgrade to HL7 2.5 or upper / future of this repo

### DISCUSSION THREAD

**Discussion by partikule** [senaranya/HL7#63] [General]
Hello,

Thanks for your adaptation of Net-HL7 library !

I have few question about the future of this project :
1. Do you plan to upgrade the HL7 version from 2.3 to any more recent version, like 2.5 ?
2. Do you plan to add more segments?

Per advance thank you for your answers.

**Comment by senaranya** [comment]
Hi,

1. I do have plans to support 2.5 alongside 2.3. I don't have a timeline yet though.
2. Of course, the more the better. Feel free to submit a PR if you need a segment right away. Else I can do it when I get a chance

**Comment by senaranya** [comment]
Thank you so much for the PR submission. Since there are quite a few additions, it might take me a little while to go through. I'll try my best to do it asap.

Meanwhile, if you can add some unit tests, that'd be awesome (unless you've already done so).

**Comment by senaranya** [comment]
Also, we need to keep 2.3 as the default version for now, and use 2.5 only when asked for. Most of the users for this package are still using v2.3
